### PR TITLE
Docs for Note Traits

### PR DESCRIPTION
**Pull request by jonathanyeung**

### CODE REVIEW

File: vault/dendron.topic.traits.api.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+id: oOIEFJc6DTgS71mmh89WQ
+title: API
+desc: ''
+updated: 1638860903879
+created: 1638844803983
+---
+
+This contains API documentation for trait behavior. 
+
+> ❗️ Note: This is an experimental API that may have breaking changes in future iterations. An increased API surface exposing more types of functionality for traits are planned in future iterations.
```
Comment (kevinslin) [inline vault/dendron.topic.traits.api.md:11]: can you add this to `#stage.germ` and add a note reference to it in here?

File: vault/dendron.topic.traits.quickstart.md [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+id: EQoaBI8A0ZcswKQC3UMpO
+title: Quickstart
+desc: ''
+updated: 1638860823247
+created: 1638843110902
+---
+
+This example will show you how you can create your own note traits and apply them to your new notes.  In this example, we'll create a trait that will modify the name and title of the note automatically when you create a new note.
+
+> ❗️ Note: This is an experimental feature that may have breaking changes in future iterations.
```
Comment (kevinslin) [inline vault/dendron.topic.traits.quickstart.md:11]: add note reference to `#stage.germ`

**Review by kevinslin — CHANGES_REQUESTED** [review]

File: vault/dendron.topic.traits.quickstart.md (hunk 2 above)
Comment (ScriptAutomate) [reply vault/dendron.topic.traits.quickstart.md:11]: I'll be making this change within the changelog update for early seed

File: vault/dendron.topic.traits.api.md (hunk 1 above)
Comment (ScriptAutomate) [reply vault/dendron.topic.traits.api.md:11]: I'll be making this change within the changelog update for early seed

Comment (ScriptAutomate) [reply vault/dendron.topic.traits.api.md:11]: See https://github.com/dendronhq/dendron-site/pull/317

File: vault/dendron.topic.traits.quickstart.md (hunk 2 above)
Comment (ScriptAutomate) [reply vault/dendron.topic.traits.quickstart.md:11]: See https://github.com/dendronhq/dendron-site/pull/317

**Pull request merged**